CHANGELOG.md

```diff
@@ -9,19 +9,32 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+### Removed
+
+### Changed
+
+### Fixed
+
+## [v0.7.0] - 2022-08-09
+
+### Added
+
 * Optional specification of a static IP address ([#509](https://github.com/quartiq/stabilizer/pull/509))
 * Temperature sensor measurement for Pounder and Stabilizer's CPU has been added [#575](https://github.com/quartiq/stabilizer/pull/575)
+* External clock support for Pounder
+* Auxiliary ADC support for Pounder
```

Comment:
```suggestion
* Auxiliary ADC support for Pounder
* Added support for STATIC_IP configuration
```

Reply:
That up there two lines above already, no?

Reply:
Woops, correct.